### PR TITLE
Feature/add GitHub actions

### PR DESCRIPTION
**Pull request by CleKraus**
Add multiple features to the workflow file such as the possibility to:

- Use different OS and Python versions
- Install dependencies through requirements file
- Test the code using pytest 
- Test coding standard through black, flake8 and isort

### CODE REVIEW

File: .github/workflows/blank.yml [hunk 1]
```diff
@@ -6,28 +6,81 @@ name: Github Actions Example
...
   # This workflow contains a single job called "build"
   build:
-    # The type of runner that the job will run on
-    runs-on: ubuntu-latest
+
+    # Define platforms and versions
+    ###############################
+    strategy:
+      matrix:
+          python-version: [3.7]
```
Comment (maximilianharr) [inline .github/workflows/blank.yml:22]:
```suggestion
          python-version: [3.8]
```

**Comment by maximilianharr** [comment]
Thanks @CleKraus 
Can you provide a quick How-To so I can make sure this PR is working on my machine as well?

**Comment by CleKraus** [comment]
@maximilianharr Thanks Max!

This is running on Github servers only, so nothing to check from your side. If you want to re-run the checks, however, you can just make a small change to this branch and push again.

**Comment by maximilianharr** [comment]
@CleKraus feel free to merge. PR approved :+1:

**Pull request merged**